defaults/security-ents.go

```diff
@@ -112,15 +123,19 @@ func securityAdminEntitlementEnforce(profile secprofile.Profile) (secprofile.Pro
 
 	capsToAdd := []types.Capability{
 		CapMacAdmin, CapMacOverride, CapDacOverride, CapDacReadSearch, CapSetpcap, CapSetfcap, CapSetuid, CapSetgid,
-		CapSysPtrace, CapFsetid, CapSysModule, CapSyslog, CapSysRawio, CapSysAdmin, CapLinuxImmutable,
+		CapSysPtrace, CapFsetid, CapSysModule, CapSyslog, CapSysRawio, CapSysAdmin, CapLinuxImmutable, CapSysBoot,
+		CapSysNice, CapSysPacct, CapSysTtyConfig, CapSysTime, CapWakeAlarm,
 	}
 	ociProfile.AddCaps(capsToAdd...)
 
 	syscallsToAllow := []types.Syscall{
-		SysPtrace, SysArchPrctl, SysPersonality, SysSetuid, SysSetgid, SysPrctl, SysMadvise,
+		SysPtrace, SysArchPrctl, SysPersonality, SysSetuid, SysSetgid, SysPrctl, SysMadvise, SysMount, SysInitModule,
+		SysFinitModule, SysSetns, SysClone, SysUnshare,
 	}
 	ociProfile.AllowSyscalls(syscallsToAllow...)
 
+	ociProfile.OCI.Linux.ReadonlyPaths = []string{}
```

Comment:
In Moby this is not defaulting to empty, this guarantees that there will be no read only paths.